### DISCUSSION TITLE
More Free Tier Limits

### DISCUSSION THREAD

**Discussion by gregg-cbs** [thunderclient/thunder-client-support#1652] [General]
Its a little unfair that all of a sudden I open vscode and I cant use thunderclient. Nothing has changed and I am getting this error. First it was the "you cant save your requests to your project anymore cause you need to pay for that now" and now its "you cannot use this because you have more than 3 collections".

At least you kept the export to postman option, i appreciate that. Maybe next week we will need to pay for that.

Pricing is fair, $3 a month isnt bad but its another subscription. Give me a one off fee and I would probably prefer to pay that. I have enough subscriptions.

For now I will head over to postman.

```text
The extension will not work if it exceeds the limits
Free Version Limits: 
Environments 0 
Collections 3 
Requests Per Collection 15

The extension will not work if any collection has more than 15 requests
Error: Collections with more than 15 requests: 
 ```

**Comment by rangav** [comment]
Hi @gregg-cbs,

Thank you for your feedback! Could you please answer the following questions:

- Do you qualify to use the free version? You can review the terms for the free version here: [Free Version Terms](https://www.thunderclient.com/terms#free-plan).
- We offer a free Community License—do you qualify for it? Check the details here: [Community License](https://www.thunderclient.com/pricing#community).

**Reply by gregg-cbs** [reply]
I do qualify for the free version. Its me and a friend working on a side hustle. I appreciate the free version or community licenses. But what im talking about in this discussion is that one day I open vscode and I can no longer view or use my requests because you can decide to change the licensing whenever you want. And it seems you do this often.

You built this extension for the community. You gave us the terms, we used it and now you are changing the terms as you please and locking us out without any notice and consulting.

**Comment by gregg-cbs** [comment]
Please can you at least allow me to view a request so I can see what url and params it is using?

Because right now you dont even allow me to open and view a saved request, you are outright blocking anything from happening.

![image](https://github.com/user-attachments/assets/1cd5bba9-066d-4745-9d21-77680c39d6ab)

**Comment by rangav** [comment]
We gave everyone 30 day notice about the changes
https://github.com/thunderclient/thunder-client-support/releases/tag/v2.29.0

Please contact us using below, We will provide alternative solutions 
https://www.thunderclient.com/contact
